### PR TITLE
Browser Monitoring Edits

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Standardizes the naming of the "RUM Browser SDK" and small copy edits.

### Motivation

#rum-docs Slack

### Additional Notes

Remember to update the branch name in `pull_config.yaml` before merging!

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/browser/collecting_browser_errors.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Source errors include code-level information about the error. More information a
 
 ## Collect errors manually
 
-Monitor handled exceptions, handled promise rejections and other errors not tracked automatically by the RUM SDK with the `addError()` API:
+Monitor handled exceptions, handled promise rejections and other errors not tracked automatically by the RUM Browser SDK with the `addError()` API:
```
Comment (alai97) [inline content/en/real_user_monitoring/browser/collecting_browser_errors.md:52]:
```suggestion
Monitor handled exceptions, handled promise rejections, and other errors not tracked automatically by the RUM Browser SDK with the `addError()` API:
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -458,18 +458,25 @@
...
     - action: pull-and-push-file
-      branch: main
+      branch: alai97/browser-monitoring-doc-update
       globs:
       - 'packages/rum/README.md'
       options:
         dest_path: '/real_user_monitoring/browser/'
         file_name: '_index.md'
         front_matters:
-          dependencies: ["https://github.com/DataDog/browser-sdk/blob/main/packages/rum/README.md"]
```
[the remark this answers is not in the document]
Comment (alai97) [reply local/bin/py/build/configurations/pull_config_preview.yaml:468]: Yes, this is just for the sake of the preview to connect the Browser Monitoring PR #1563 with!

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 3]
```diff
@@ -458,18 +458,25 @@
   - repo_name: browser-sdk
     contents:
     - action: pull-and-push-file
-      branch: main
+      branch: alai97/browser-monitoring-doc-update
```
Comment (alai97) [inline local/bin/py/build/configurations/pull_config_preview.yaml:461]:
```suggestion
      branch: main
```

**Review by apigirl — APPROVED** [review]

File: local/bin/py/build/configurations/pull_config_preview.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (apigirl) [reply local/bin/py/build/configurations/pull_config_preview.yaml:468]: ❗ **Information** So this is the `pull_config_preview.yaml`, the main site is pulled from the `pull_config.yaml` file. We've split them out to keep this problem from happening! So you should definitely return the branch to `main`, but even if it is merged in as-is, it won't be for production builds, just the preview sites. You should also always make sure to update both files with final changes.

**Pull request merged**